src/main/resources/io/snyk/jenkins/SnykStepBuilder/config.jelly

```diff
@@ -45,30 +45,24 @@
     <f:entry title="Project name" field="projectName">
       <f:textbox/>
     </f:entry>
-  </f:section>
-
-  <!-- advanced configuration -->
-  <f:section title="">
-    <f:advanced>
```

Comment:
Removed the "Advanced" toggle as time and time again it causes issues with hidden configuration. Users aren't aware of it as it's a small button. If you have configured an "advanced" field, next time you open the config page, it's hidden again so you might not realise you've configured one of the fields under it.

It's just two fields anyway and not really advanced.